### PR TITLE
Improved Contributing Guidelines

### PR DESCRIPTION
**Pull request by amsos**
Added some improved Contributing Guidlines, Template for Issues, and a TOC for the Readme (I think tthat's it).

❗️One callout question, I linked to the TG group for people to use if they wanted to join the conversation.  Not sure if you wanted that or not.

hope this is helpful

😘

### CODE REVIEW

**Comment by amsos** [comment]
Ohhh im also looking at the diffs now.  you'll see a lot of changes on the readme because of some autoformat rules.

let me know if you have preferred formating rules you want me to rerun this with.

**Comment by amsos** [comment]
👀  @slgraham @nintynick

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Feature Request
+about: New ideas are fun
+---
+
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < <
+v Lets brainstorm new ideas together
+v Note: If you're proposal isn't formatted correctly, we will drop it :)
```
Comment (slgraham) [inline .github/ISSUE_TEMPLATE/feature_request.md:8]: "you're" should be "your"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+<!-- 3 sentences on the why -->
+
+## Steps to Reproduce
+
+<!-- Most important piece, please be clear -->
+
+---
+
+## You're interest in fixing
```
Comment (slgraham) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]: "you're" should be "your"

**Comment by slgraham** [comment]
@amsos thanks for these suggestions! Overall I like them. A couple notes before merging:

- I added in some markdown linting, both in the repo itself with the `.markdownlintrc` file and with a github action check. Please rebase to include those. That should help with the formatting consistency.
- There are a couple instances of "you're" vs. "your", noted in in-line comments
- Please do take out the reference to tg. We'll add in some contact info when we're ready.

**Pull request closed without merging**